Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanel.mm

```diff
@@ -268,13 +281,24 @@ - (void)setDelegate:(id<_WKWebAuthenticationPanelDelegate>)delegate
     }];
     if (byRpId)
         [query setObject:byRpId forKey:bridge_cast(kSecAttrLabel)];
-    if (byCredentialId)
-        [query setObject:byCredentialId forKey:bridge_cast(kSecAttrApplicationLabel)];
+    if (byCredentialId) {
+        if (useAlternateKeychainAttribute)
+            [query setObject:byCredentialId forKey:bridge_cast(kSecAttrAlias)];
+        else
+            [query setObject:byCredentialId forKey:bridge_cast(kSecAttrApplicationLabel)];
```

Comment:
You repeat this block of code a bunch of times, too. Why not use something like:

```
CFStringRef credentialKey = useAlternateKeychainAttribute ? kSecAttrAlias : kSecAttrApplicationLabel;
[query setObject:byCredentialId forKey:bridge_cast(credentialKey)];
```

Reply:
I've applied this to all the places where I add one key or the other to a dictionary